tests/test_integration.py

```diff
@@ -29,28 +30,40 @@ def test_train_and_run(
         str(mgf_small),
         "--config",
         tiny_config,
-        "--output",
-        str(tmp_path / "train"),
+        "--output_dir",
+        str(tmp_path),
+        "--output_root",
+        "train",
         str(mgf_small),  # The training files.
     ]
 
     result = run(train_args)
-    model_file = tmp_path / "epoch=19-step=20.ckpt"
-    best_model = tmp_path / "best.ckpt"
+    model_file = tmp_path / "train.epoch=19-step=20.ckpt"
+    best_model = tmp_path / "train.best.ckpt"
     assert result.exit_code == 0
     assert model_file.exists()
     assert best_model.exists()
 
-    # Finally try predicting:
-    output_filename = tmp_path / "test.mztab"
+    # Check that re-running train fails due to no overwrite
```

Comment:
**suggestion:** I don't think we need to cover this in the integration test, it's already verified in `test_runner` where the relevant error is actually thrown.